javascript/ql/lib/semmle/javascript/frameworks/Templating.qll

```diff
@@ -580,6 +580,22 @@ module Templating {
     override string getAPackageName() { result = "ejs" }
   }
 
+  /**
+   * doT-style syntax, using `{{! }}` for safe interpolation, and `{{= }}` for
+   * unsafe interpolation.
```

Comment:
Looks like the two escaping mechanisms were flipped around, either in the comment or in the implementation.

- The comment says `{{! }}` is safe, but the implementation treats it as raw (i.e. unsafe)
- The comment says `{{= }}` is unsafe, but the implementation treats it as escaping (i.e. safe, in most contexts).

Reply:
Oops, sorry, the model is right, `{{! }}` is for unsafe (raw) interpolation.